website/docs/models/index.md

```diff
@@ -154,10 +187,31 @@ nlp.enable_pipe("senter")
 The `senter` component is ~10&times; faster than the parser and more accurate
 than the rule-based `sentencizer`.
 
+#### Switch from trainable lemmatizer to default lemmatizer
+
+Since v3.3, a number of pipelines use a trainable lemmatizer. You can check whether
+the lemmatizer is trainable:
+
+```python
+nlp = spacy.load("de_core_web_sm")
+assert nlp.get_pipe("lemmatizer").is_trainable
+```
+
+If you'd like to switch to a non-trainable lemmatizer that's similar to v3.2 or
+earlier, you can replace the trainable lemmatizer with the default non-trainable
+lemmatizer:
+
+```python
+# Requirements: pip install spacy-lookups-data
+nlp = spacy.load("de_core_web_sm")
+nlp.remove_pipe("lemmatizer")
+nlp.add_pipe("lemmatizer").initialize()
```

Comment:
Hmmm the naming is still a bit confusing here, I think. Should we clarify this in the docs that they are from different factories and that the trainable one has been renamed for convenience?

Maybe it helps if we also show how to switch from a non-trainable one to a trainable one?

Reply:
Maybe just clarify that this adds the non-trainable lemmatizer from the language defaults.

I don't want to try how to switch to a trainable lemmatizer in an existing pipeline because there are too details (frozen, which tok2vec, etc.).